packages/manager/modules/sms/src/index.js

```diff
@@ -14,7 +16,10 @@ angular
       $stateProvider
         .state('sms', {
           url: '/sms',
-          abstract: true,
+          redirectTo: 'sms.index',
+          resolve: {
+            breadcrumb: () => PRODUCT_NAME,
```

Comment:
don't you want to return $translate('sms_sms_breadcrumb') here ?

also i don't think it's necessary to create a constant for 'SMS' value given the module name is 'sms', a string literal would be completely fine imo

Reply:
This means that will have to load from another module or add a translation, I'm not sure it makes sense given the product name is static